src/mlpack/methods/naive_bayes/nbc_main.cpp

```diff
@@ -112,20 +119,27 @@ PARAM_FLAG("incremental_variance", "The variance of each class will be "
 
 // Test parameters.
 PARAM_MATRIX_IN("test", "A matrix containing the test set.", "T");
+// The parameter 'output' is deprecated and will be removed in mlpack 4.0.
 PARAM_UROW_OUT("output", "The matrix in which the predicted labels for the"
     " test set will be written.", "o");
+PARAM_UROW_OUT("predictions", "The matrix in which the predicted labels for the"
+    " test set will be written.", "a");
+PARAM_MATRIX_OUT("probabilities", "The matrix in which the predicted"
+    " probability of labels for the test set will be written.", "p");
+// The parameter 'output_probs' is deprecated and can be removed in mlpack 4.0.
 PARAM_MATRIX_OUT("output_probs", "The matrix in which the predicted probability"
-    " of labels for the test set will be written.", "p");
+    " of labels for the test set will be written.", "s");
 
 static void mlpackMain()
 {
   // Check input parameters.
   RequireOnlyOnePassed({ "training", "input_model" }, true);
   ReportIgnoredParam({{ "training", false }}, "labels");
   ReportIgnoredParam({{ "training", false }}, "incremental_variance");
-  RequireAtLeastOnePassed({ "output", "output_model", "output_probs" }, false,
-      "no output will be saved");
+  RequireAtLeastOnePassed({ "output", "predictions", "output_model",
+     "output_probs", "probabilities" }, false, "no output will be saved");
```

Comment:
There's a little style issue here... a wrapped line should be indented twice (four spaces) but this looks like 3 spaces.